CHANGELOG.md

```diff
@@ -4,44 +4,56 @@
 
 ### New features
 
-#### Create custom width container classes
+#### Set a specific width for wrappers
 
-You can now create custom page width container classes using the `govuk-width-container` mixin. You do this by passing in the required maximum width of the container.
+You can now:
+
+- create your own Sass classes with specific widths
+- add those classes to the `.govuk-width-container` wrapper using Nunjucks
+
+This means you can set specific widths for the:
+
+- [page wrapper](https://design-system.service.gov.uk/styles/layout/#page-wrappers)
+- [header component](https://design-system.service.gov.uk/components/header/)
+- [footer component](https://design-system.service.gov.uk/components/footer/)
+
+Include the `govuk-width-container` mixin inside your Sass class definition, and pass in the width in pixels.
 
 For example:
 
 ```scss
-.app-width-container--wide {
+.YOURAPP-width-container--wide {
```

Comment:
I think it's fine to keep this as 'app', not sure if it's required to explain namespaces here, since for most people 'app' will be fine.

Reply:
@hannalaakso and I discussed this quite a bit - there's a big risk of confusion here (I've tripped up on it myself!) because there's no clear distinction between placeholder names and copy-and-paste code. It's especially problematic here because you have an `app` folder in Frontend. So it's worth clarifying - and it won't hold up users who already know about namespaces.

Reply:
>  big risk of confusion

There's no risk that I know of people copying 'app-width-container--wide' into their codebase, and this is what we expect a large majority of people to do, there's only a small part of our user base that require something different to 'app'.

Reply:
This mirrors the precedent set in the https://design-system.service.gov.uk/get-started/extending-and-modifying-components/ guidance where we use 'app-' classes

Reply:
No probs, but if we use `app` I'd like to be specific in https://github.com/alphagov/govuk-frontend/pull/1637#discussion_r345129313 by naming the element of the Sass file.